src/NuGet.Core/NuGet.CommandLine.XPlat/RestoreCommand.cs

```diff
@@ -63,6 +63,11 @@ internal static class RestoreCommand
                     Strings.Restore_Arg_ProjectName_Description,
                     multipleValues: true);
 
+                var ignoreFailedSources = restore.Option(
+                    "-ignoreFailedSources",
```

Comment:
We should use the same flag name that DNU had ``--ignore-failed-sources``

Also note that xplat uses two dashes instead of one.